chia/wallet/wallet_node.py

```diff
@@ -979,7 +979,13 @@ async def fetch_and_validate_the_weight_proof(
         assert self.wallet_state_manager.weight_proof_handler is not None
 
         weight_request = RequestProofOfWeight(peak.height, peak.header_hash)
-        weight_proof_response: RespondProofOfWeight = await peer.request_proof_of_weight(weight_request, timeout=60)
+        wp_timeout = 360
+        if "weight_proof_timeout" in self.config:
+            wp_timeout = self.config["weight_proof_timeout"]
```

Comment:
```suggestion
wp_timeout = self.config.get("weight_proof_timeout", 300)
```

But, shouldn't this be done at some startup point rather than in the middle of everything?  I thought we processed config in `.__init__()` or such generally.